pkg/apis/proxy/v1alpha1/managedproxyconfiguration_types.go

```diff
@@ -219,6 +219,11 @@ type ManagedProxyConfigurationProxyServerEntrypoint struct {
 	// `hostname` points to a fixed hostname for serving agents' handshakes.
 	// +optional
 	Hostname *EntryPointHostname `json:"hostname"`
+
+	// `port` points to the port proxy-agent should access
+	// +optional
+	// +kubebuilder:default=8091
```

Comment:
to keep the backward-compatbility, lets set the default to 443, also add a validation marker that asserts positive value

Reply:
The thing is we need to set it to 8091 to keep the backward compatibility because ANP uses 8091 as the default agent-port. If we set the default value to 443, the new version could not access to the previous one.

https://github.com/kubernetes-sigs/apiserver-network-proxy/blob/d2eec1ec3ac33f683c81a950f171caab568c0250/cmd/server/app/options/options.go#L308